submissions/m-ruslan/html-css-popup/app.css

```diff
@@ -0,0 +1,248 @@
+/* main styles */
+
+body {
+  background-color: #f0eee8;
+  font-family: sans-serif;
+  margin: 0;
+}
+
+nav {
+  width: 550px;
+  float: right;
+  background-color: white;
+  height: 37px;
+  margin: 20px;
+  border-radius: 5px;
+}
+
+.nav-container {
+  float: right;
+  width: 300px;
+  height: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+input[type="checkbox"] {
+  position: absolute;
+  top: -9999px;
+  left: -9999px;
+}
+
+.nav-el > a {
```

Comment:
Better way to use classes for styling. 
Smth like `nav-el__item` (if you use [BEM](https://ru.bem.info/methodology/)).
It will make your code more universal (for example if you change `a` to `button` or `div`).

Reply:
Thank you, I've tried to apply basic rules of BEM, please review my updated code.